packages/cli/src/transforms/transform.assets.js

```diff
@@ -0,0 +1,59 @@
+const path = require('path');
```

Comment:
it also does fonts

Reply:
then we should probably have two then (fonts and images) :)

Reply:
not a big deal for now, will add it as a follow up, but in my mind, I could even see some of these transforms getting more granular, e.g.
- transform.html.import-map.js
- transform.html.livereload.js
- etc

but again, one step at a time.  I had some ideas on how to scale this out a bit so will take a stab at this API myself to help contribute to the design instead of just pontificating.  😄